src/Qir/Tests/QIR-dynamic/qsharp/qir-test-assert.qs

```diff
@@ -1,15 +1,235 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+// https://docs.microsoft.com/en-us/azure/quantum/concepts-the-qubit#single-qubit-operations    Qubit and Operations
+// https://docs.microsoft.com/en-us/azure/quantum/user-guide/libraries/standard/prelude         Operation Details, Multi-Qubit Operations
+
 namespace Microsoft.Quantum.Testing.QIR  {
 
     open Microsoft.Quantum.Diagnostics;
+    open Microsoft.Quantum.Intrinsic;
+
+    // |0> -> |1>
+    @EntryPoint()
+    operation AsrtMeasAlloc1OKTest() : Unit {
```

Comment:
Nit, but we've generally not used this form of abbreviation in operation names; I'd suggest expanding `Asrt` in this case.

Reply:
Re4places `Asrt` with `Assert`.